blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -103,45 +108,44 @@ export class Activity extends Component {
         return (<main>
             <article className="activity">
                 { showRunButton &&
-                <RunButton
-                  runnable={pipeline}
-                  latestRun={latestRun}
-                  buttonType="run-only"
-                  onNavigation={onNavigation}
-                  innerButtonClasses="btn-secondary"
-                />
+                    <ParametersRunButton
```

Comment:
@michaelneale yes it contains the regular `RunButton` and shows it when no `parameters` are detected. 

Regarding naming `ParametersAwareRunButton` had been to long but that would be the most specific name.